chart/compass/templates/registration-job.yaml

```diff
@@ -60,7 +60,8 @@ spec:
               TOKEN_PAYLOAD='{"scopes": "'${SCOPES}'","tenant":"{{ .Values.global.defaultTenant }}"}'
```

Comment:
Why is it moved from `provisioner` chart? why bash?

Reply:
We have agreed with PK to move it to the chart's root, use its codebase and only create a separate secret for KEB. I would prefer to have it written in Go.